package.json

```diff
@@ -87,15 +87,15 @@
     "tslib": "^1.11.1"
   },
   "devDependencies": {
-    "@pdf-lib/fontkit": "^1.0.0",
+    "@pdf-lib/fontkit": "^1.1.0",
     "@rollup/plugin-commonjs": "^13.0.0",
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^8.0.1",
     "@types/jest": "^26.0.0",
     "@types/node-fetch": "^2.5.7",
     "@types/pako": "^1.0.1",
     "@zerollup/ts-transform-paths": "^1.7.18",
-    "downlevel-dts": "^0.5.0",
+    "downlevel-dts": "^0.6.0",
```

Comment:
Does this need to be updated as part of this PR?

Reply:
To be honest I don't know, I believe this was caused by an `audit fix`.
I'm also not super familiar writing tests either, but I can give it a go.

Reply:
In that case I'd like to undo this change. If the dependency needs to be updated, I'd prefer that to be a separate PR. But I don't think it's necessary for the changes you're making anyways.

Reply:
Also, with respect to testing, no worries if you're new to it! I'd recommend you read over the [`CONTRIBUTING.md`](https://github.com/Hopding/pdf-lib/blob/master/CONTRIBUTING.md) file. All contributors should read it. There are sections explaining how to run the [unit tests](https://github.com/Hopding/pdf-lib/blob/master/CONTRIBUTING.md#running-the-unit-tests), [integration tests](https://github.com/Hopding/pdf-lib/blob/master/CONTRIBUTING.md#running-the-integration-tests) (though you don't need to worry about running the React Native tests), and [running the typechecker](https://github.com/Hopding/pdf-lib/blob/master/CONTRIBUTING.md#running-the-type-checker) (amongst other important things). Let me know if you have any questions.